RUFAS/input_manager.py

```diff
@@ -848,6 +1066,46 @@ def _validate_dict_element(  # noqa
                     element_counter_and_validity["is_valid"] = False
                 return element_counter_and_validity
 
+    def _validate_json_element(  # noqa
```

Comment:
```suggestion
    def _validate_json_element(
```
flake8 has no issues with this method, why is there noqa here?

Reply:
Removed now.